crates/sdk/src/lib.rs

```diff
@@ -1085,11 +1085,11 @@ pub mod testing {
 
     prop_compose! {
         /// Generate an arbitrary header
-        pub fn arb_header()(
+        pub fn arb_header(cmt_count: impl Into<SizeRange>,)(
             chain_id in arb_chain_id(),
             expiration in option::of(arb_date_time_utc()),
             timestamp in arb_date_time_utc(),
-            batch in arb_tx_commitments(1..10),
+            batch in arb_tx_commitments(cmt_count),
```

Comment:
Nice one here 👍🏻 , we were previously generating up to 10 headers and the other `arb_transaction` generators were only appending data for a single inner tx which was strange